packages/ffe-form-react/src/RadioBlock.js

```diff
@@ -11,6 +11,12 @@ class RadioBlock extends Component {
             checked,
             children,
             className,
+            /* Support for the dark theme has not been added for this component
+             * but since we're passing props onwards to DOM nodes we need to make
+             * sure we don't pass this as part of "inputProps" since it's not being
+             * used but can still be passed down from a parent `RadioButtonInputGroup`.
+             */
+            dark, //eslint-disable-line
```

Comment:
This should then be added as an optional variable to ts definitions, otherwise it will give an error.
If we want to pass everything on, we can use     [propName: string]: any;, but I would prefer to just add the parameter with boolean and add the same comment together with the type definition

Reply:
Ah ok. It makes sense that it would throw, since that's what passing it onwards does (React sees an all-lowercase prop and assumes it's meant as a custom attribute but those can only be strings), I just totally forgot that it's not something we're passing manually. Will fix.